qui/tray/devices.py

```diff
@@ -120,7 +120,17 @@ def add_vm(self, _, obj_path):
             self.attached_item = menu_item
 
         self.menu_items[str(obj_path)] = menu_item
-        self.append(menu_item)
+        
+        # sort function
+        position=0
+        list_tmp={}
+        for i in self.menu_items:
+            list_tmp[str(self.menu_items[i].vm)]=i
+        for i in sorted(list_tmp):
+            if i<str(vm["name"]):
+                position +=1
```

Comment:
Why not iterate over menu items directly? I.e. without `list_tmp` variable? Something like:
```python
for item_index in self_menu_items:
    if str(self.menu_items[item_index].vm) < str(vm["name"]):
        position += 1
```
Also, code style: please add spaces around operators (`=`, `<` etc).

Reply:
you're right, I corrected
thank

Reply:
This still use temporary list (so, traverse the whole list of VMs twice).

Reply:
I modified, how I valid